Source/WebKit/Scripts/webkit/messages.py

```diff
@@ -376,13 +373,16 @@ def types_that_cannot_be_forward_declared():
         'WebKit::ImageBufferBackendHandle',
         'WebKit::LayerHostingContextID',
         'WebKit::LegacyCustomProtocolID',
+        'WebKit::RemoteMediaSourceIdentifier',
+        'WebKit::RemoteSourceBufferIdentifier',
         'WebKit::RemoteVideoFrameWriteReference',
         'WebKit::RemoteVideoFrameReadReference',
         'WebKit::RenderingUpdateID',
         'WebKit::TextCheckerRequestID',
         'WebKit::TransactionID',
         'WebKit::WCLayerTreeHostIdentifier',
         'WebKit::WCContentBufferIdentifier',
+        'WebKit::XRDeviceIdentifier',
```

Comment:
This change is a bit of a stab in the dark and could use review from @achristensen07. The `serialized_identifiers()` generator was added recently in https://github.com/WebKit/WebKit/commit/d32d14b0307fbac280cfb1b55835657d63daef7d and doesn't have a way to conditionally reference these types. I'm not sure why the tvOS 16 SDK would behave differently here, as these symbols should _also_ be missing in the current tvOS EWS build. But this is what I needed to get my build working.

Reply:
EWS likely does not build debug / release+asan for tvOS.
The IPC test api feature is enabled only on these.